tests/Microsoft.Identity.Test.LabInfrastructure/KeyVaultSecretsProvider.cs

```diff
@@ -51,7 +51,8 @@ public class KeyVaultSecretsProvider : IDisposable
 
         private KeyVaultConfiguration _config;
 
-        private readonly string _keyVaultClientID = "ebe49c8f-61de-4357-9194-7a786f6402b4";
+        private readonly string _keyVaultConfidentialClientID = "ebe49c8f-61de-4357-9194-7a786f6402b4";
```

Comment:
use `const` for these. const is a superset of readonly + static. Naming convention is capital case, e.g. KeyVaultClientId